src/CLI_Installer/Product.wxs

```diff
@@ -20,41 +20,35 @@ Licensed under the MIT license. See LICENSE file in the project root for full li
 
     <Property Id="NETCORERUNTIMEFOUNDX86">
       <DirectorySearch Id="NetCoreDirectoryFoundx86" Path="[ProgramFilesFolder]dotnet" >
-        <FileSearch Name="dotnet.exe" MinVersion="3.0"/>
+        <FileSearch Name="dotnet.exe" MinVersion="3.1"/>
       </DirectorySearch>
     </Property>
 
     <Property Id="NETCORERUNTIMEFOUNDX64">
       <DirectorySearch Id="NetCoreDirectoryFoundx64" Path="[ProgramFiles64Folder]dotnet" >
-        <FileSearch Name="dotnet.exe" MinVersion="3.0"/>
+        <FileSearch Name="dotnet.exe" MinVersion="3.1"/>
       </DirectorySearch>
     </Property>
 
     <MediaTemplate EmbedCab="yes" />
 
     <Feature Id="ProductFeature" Title="Axe.Windows CLI" Level="1">
       <ComponentGroupRef Id="ProductComponents" />
-      <ComponentGroupRef Id="NetCoreApp20Components" />
-      <ComponentGroupRef Id="NetCoreApp21Components" />
       <ComponentGroupRef Id="NetCoreApp30Components" />
-      <ComponentGroupRef Id="NetStandard20Components" />
     </Feature>
 
-    <Condition Message="[ProductName] requires .NET Core Runtime 3.0 or newer. Please visit https://dotnet.microsoft.com/download/dotnet-core">
+    <Condition Message="[ProductName] requires .NET Core Runtime 3.1 or newer. Please visit https://dotnet.microsoft.com/download/dotnet-core">
       <![CDATA[Installed OR NETCORERUNTIMEFOUNDX64 OR NETCORERUNTIMEFOUNDX86]]>
     </Condition>
 
     <Directory Id="TARGETDIR" Name="SourceDir" >
       <Directory Id="ProgramFilesFolder" >
         <Directory Id="AxeWindowsCLIFolder" Name ="AxeWindowsCLI" >
-          <Directory Id="INSTALLFOLDER" Name="1.0" >
+          <Directory Id="INSTALLFOLDER" Name="$(var.SemVer)" >
```

Comment:
This isn't really related to the .NET Core 3.1 changes--could be a separate PR if desired